source/settings/settings-tasks.txt

```diff
@@ -0,0 +1,36 @@
+=====
+Tasks
```

Comment:
[suggestion] I think this was a good initial idea, but after giving it some thought, I don't think we should hide these tasks under a folder. I'd suggest allowing them to live under "Customize Compass."

Reply:
Noted. I agree that on second look the folder just creates another unnecessary layer that users would have to click through. On the other hand, do you think it'd make the "Customize Compass" folder seem a bit cluttered with the reference pages and task pages all grouped under one folder?

Reply:
I agree that there's a lot here, but I think it's okay because it does reflect the level of customization available.